### PR TITLE
Webserver

### PR DESCRIPTION
**Pull request by alexanderFell**
I added a simple webserver interface to hyperion and hyperion-audio-effects solving issue #13

### CODE REVIEW

**Comment by RanzQ** [comment]
Could you paste some screenshot of this? Did you know that for hyperion a web UI already exists: https://hyperion-project.org/wiki/WebApp-Remote

Thanks for your contribution!

**Comment by alexanderFell** [comment]
I saw that, but the browser version does not work in my case and I do not want to install any app (e.g. for chrome). I do not have Chrome installed on all my machines. Apart from that, I wanted to control the audio effects as well :)

Here are some screenshots. As I said, just very simple, but it serves it purpose.

screenshots_2016-09-09-20-28-07
screenshots_2016-09-09-20-28-18
screenshots_2016-09-09-20-28-34

**Comment by redPanther** [comment]
just a note: in next major version of hyperion, hyperion itself will  have an webserver ... no additional software have to be installed for that. (see https://github.com/hyperion-project/hyperion.ng its the development version, currently und heavy development)

File: effects/allEffectsRandom.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name" : "Random Effects",
+    "script" : "allEffectsRandom.py",
```
Comment (RanzQ) [inline effects/allEffectsRandom.json]: I think snake_case is more pythonic, so this should be `all_effects_random.py`.

File: webserver.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+		self.getFile('style.css')
+		self.sendFooter();
+
+
+
+	def musicEffects(self):
+		effects = []
+		for effect in os.listdir(os.path.dirname(os.path.abspath(__file__))+"/effects"):
+			if effect.endswith(".json"):
+				effects.append(effect[:-5])
```
Comment (RanzQ) [inline webserver.py]: Would be better to read in the `json` and use `script` field for commands and `name` field for UI.

**Comment by RanzQ** [comment]
Closing this due to inactivity. A couple of things, I wasn't happy with the UI looks and I asked for lint check.

**Pull request closed without merging**